cognite_toolkit/_cdf_tk/data_classes/_module_toml.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+import sys
+from dataclasses import dataclass, field
+from pathlib import Path
+
+if sys.version_info >= (3, 11):
+    import toml
+else:
+    import tomli as toml
+
+
+@dataclass(frozen=True)
+class ModuleToml:
+    version: str
```

Comment:
Should modules have a version? I think that is unnecessary complexity. Instead, all builtin modules have shared version.